eng/pipelines/templates/jobs/apiview-review-gen-python.yml

```diff
@@ -0,0 +1,59 @@
+pr: none
+
+trigger: none
+
+pool:
+  name: azsdk-pool-mms-ubuntu-2004-general
+  vmImage: MMSUbuntu20.04
+
+parameters:
+  - name: Reviews
+    type: string
+    default: ''  
+  - name: APIViewURL
+    type: string
+    default: '' 
+  - name: StorageContainerUrl
+    type: string
+    default: ''
+
+variables:
+  NugetSecurityAnalysisWarningLevel: none
+  skipComponentGovernanceDetection: true
+  PythonIndexUrl: 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/'
+  PythonVersion: '3.10.2'
+  ApiStubVersion: '0.2.11'
+
+jobs:
+- job: CreateReviewCodeFile
+  displayName: Create APIView code file
```

Comment:
I would probably give this a language specific name so it is easier to know which language we are processing.

Reply:
done